api/app/controllers/spree/api/v2/storefront/wished_products_controller.rb

```diff
@@ -3,15 +3,14 @@ module Api
     module V2
       module Storefront
         class WishedProductsController < ::Spree::Api::V2::ResourceController
-
           def create
             spree_authorize! :create, Spree::WishedProduct
             spree_authorize! :update, wishlist
 
             wished_product = Spree::WishedProduct.new(wished_product_attributes)
 
             if wishlist.include? params[:wished_product][:variant_id]
-              wished_product = wishlist.wished_products.detect {|wp| wp.variant_id == params[:wished_product][:variant_id].to_i }
+              wished_product = wishlist.wished_products.detect { |wp| wp.variant_id == params[:wished_product][:variant_id].to_i }
```

Comment:
```suggestion
              wished_product = wishlist.wished_products.detect { |wp| wp.variant_id.to_s == params[:wished_product][:variant_id] }
```

in future we will move to UUID so we cannot assume that IDs will be integers

also let's use `any?` or `exists?` rather than creating a collection of objects in memory and iterating them

Reply:
Still to do